### ISSUE TITLE
Search Autocompletion

### ISSUE THREAD

**Issue by WizardOfMenlo** [jackjdleslie/LatinDiachronicFrontendOld#2]
I believe it would be extremely helpful if we could implement auto-completion. For the authors comparison it should be relatively straight forward, while the word search could prove more challenging.
 I will be checking what changes are required on the back end to allow this.

**Comment by jackjdleslie** [comment]
Yeah good shout, is there a limit to be enforced on the number of authors you can enter?

**Comment by WizardOfMenlo** [comment]
Well, we currently have ~350 authors, so that will be one of the limits, but theoretically no. Users are not going to be wanting to do that particularly often anyways, as ```(∩ A_i) / (U B_i) ``` should have the first term tending to ∅ as the number of ```A_i```s increases.

**Comment by jackjdleslie** [comment]
Added basic version of this in commit 61c50c3, see https://latin.netlify.com/search/intersection

**Comment by jackjdleslie** [comment]
Need to add fuzzy search

**Comment by jackjdleslie** [comment]
The latest commit 9fb1839 simulates a fuzzy search, might suffice?
